static.go

```diff
@@ -1,6 +1,7 @@
 package main
 
-var LOGINPAGE = `{{define "login"}}
+var TEMPLATES = `
```

Comment:
There are four templates: `login`, `register`, `invalid` (for failed automatic login with username and verification key), and `unauthorized` (for unauthorized service).